### ISSUE TITLE
not able to run the test pipeline

### ISSUE THREAD

**Issue by fmeer-code** [uhlmanngroup/muMatch#4]
Hi,

After creating the environment and pip install everything as described in the readme, I get the following error when I call the function run_microMatch_test():

![Screenshot 2022-10-07 at 11 25 42](https://user-images.githubusercontent.com/26184752/194521086-8ce7836f-13c1-47cc-8417-9bf29b6e4cf8.png)

complete error file:

[error.txt](https://github.com/uhlmanngroup/muMatch/files/9732606/error.txt)

Does anyone get the same error or knows how to fix this?

**Comment by jjk01** [comment]
Hello,
Sorry for the late reply to this. This is a run-time error in the neural network (I assume during training). You might try playing with the parameters of the network (learning rate etc) and also the ls_regularizer term in line 17 of /main/micro_match/preprocessing/deep_functional_maps/operations.py

**Comment by Euan-Mac** [comment]
Hi,

I am facing the same issue now. I wonder if you ever resolved this @fmeer-code ? I have upped the l2_regularizer as suggested (increased its value from 1e-2, to 1e-1) and the epochs go further but still eventually seem to blow up.
